Wikipedia/Code/ReadingThemesControlsViewController.swift

```diff
@@ -79,6 +79,25 @@ open class ReadingThemesControlsViewController: UIViewController, AnalyticsConte
         NSObject.cancelPreviousPerformRequests(withTarget: self)
     }
     
+    open override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        let tap = UITapGestureRecognizer(target: self, action: #selector(self.handleTap(_:)))
+        slider.addGestureRecognizer(tap)
+    }
+    
+    func handleTap(_ sender: UIGestureRecognizer) {
+        let pointTapped: CGPoint = sender.location(in: slider)
+        
+        let widthOfSlider: CGFloat = slider.bounds.size.width
+        let newValue = pointTapped.x * (CGFloat(slider.numberOfSteps) / widthOfSlider)
+
+        slider.value = Int(newValue)
```

Comment:
With `rounded()` not all steps are being respected 🤔
![rounded](https://user-images.githubusercontent.com/9299317/29038989-9583ca4a-7b77-11e7-80fb-31bd07f83537.gif)